### DISCUSSION TITLE
Convert to This Supported Format

### DISCUSSION THREAD

**Discussion by Jay123210599** [mifi/lossless-cut#2156] [Q&A]
I have chosen this format for unsupported formats. Now I want to keep this option every time I open one, no matter how many times I restart the tool. How do I do that?
![Screenshot (1118)](https://github.com/user-attachments/assets/1de92b50-f153-407a-b2e6-fbc81d94a30d)

**Comment by mifi** [comment]
That’s not possible

**Reply by Jay123210599** [reply]
why?

**Reply by mifi** [reply]
Because it will only remember the setting in RAM. RAM gets cleared when you quit losslesscut. If you want the setting to be remembered, then refrain from closing losslesscut. See also #371
